newdle/core/app.py

```diff
@@ -81,6 +83,34 @@ def _handle_exception(exc):
         return jsonify(error='internal_error'), 500
 
 
+def _configure_cli(app):
+    @app.cli.command('cleanup_newdles')
+    def _cleanup_newdles():
+        """Removes old newdles from the database."""
+        last_activity_cleanup_interval = timedelta(
+            days=app.config['LAST_ACTIVITY_CLEANUP_INTERVAL']
+        )
+        final_date_cleanup_interval = timedelta(
+            days=app.config['FINAL_DATE_CLEANUP_INTERVAL']
+        )
+        now = datetime.utcnow()
+        final_newdles_to_delete = Newdle.query.filter(
+            Newdle.final_dt.isnot(None),
+            Newdle.final_dt + final_date_cleanup_interval <= now,
```

Comment:
`final_dt` isn't UTC but in the newdle's timezone. Not that it really matters here... 

I think something like this should work to convert it to UTC (untested):

    final_dt_utc = func.timezone('UTC', func.timezone(Newdletimezone, Newdle.final_dt))

and then use that in the query here.

Reply:
Is it worth complicating it then or just leaving it as is?

Reply:
I'd be fine with leaving it as-is and just adding a comment

```
# XXX: This does not take newdle timezones into account, but a
# few hours are not significant here
``

or something like that